modules/shared/components/molecules/TrendeingQuestion/TrendingQuestion.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import Link from 'next/link';
+import ImagePoll from './icons/imagePoll.svg';
+import TextPoll from './icons/textPoll.svg';
+import MiniSurvey from './icons/miniSurvey.svg';
+import type { ITrendingQuestion } from './ITrendingQuestion';
+
+export const truncate = (str: string): string => {
+  const limit = 30;
+  return str.length > limit ? `${str.substr(0, limit - 1)}...` : str;
+};
+
+const TrendingQuestion: FC<ITrendingQuestion.IProps> = ({
+  text,
+  type,
+}): ReactElement => {
+  return (
+    <div className="flex">
+      {type === 'Mini Survey' && <MiniSurvey />}
+      {type === 'Text Poll' && <TextPoll />}
+      {type === 'Image Poll' && <ImagePoll />}
+      <div className="ml-4">
```

Comment:
use `title` attribute and this div and pass the imgCaption value to it so when the user hovers on the question he will see the full text